core/src/main/java/nl/nn/adapterframework/jdbc/JdbcFacade.java

```diff
@@ -123,7 +123,7 @@ protected DataSource getDatasource() throws JdbcException {
 	public String getDatasourceInfo() throws JdbcException {
 		String dsinfo=null;
 		if(getDatasource() instanceof TransactionalDbmsSupportAwareDataSourceProxy) {
-			return ((TransactionalDbmsSupportAwareDataSourceProxy) getDatasource()).getInfo();
+			return ((TransactionalDbmsSupportAwareDataSourceProxy) getDatasource()).toString();
```

Comment:
Dit is niet handig, de tostring geeft meer/minder data terug afhankelijk van het log level.